mappings/net/minecraft/client/render/entity/EntityRenderDispatcher.mapping

```diff
@@ -72,9 +82,9 @@ CLASS net/minecraft/class_898 net/minecraft/client/render/entity/EntityRenderDis
 	METHOD method_3950 shouldRender (Lnet/minecraft/class_1297;Lnet/minecraft/class_4604;DDD)Z
 		ARG 1 entity
 		ARG 2 frustum
-		ARG 3 cameraX
-		ARG 5 cameraY
-		ARG 7 cameraZ
+		ARG 3 x
+		ARG 5 y
+		ARG 7 z
```

Comment:
So this is the position at which the entity should render? If not, you should keep the `camera` prefix.